FinaliseWebpages.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/python3
+
+import os
+import WebpageFunctions
+
+#Grab environment variables, if not exisiting, replace with default value.
+ValidationPath = os.getenv("ValidationPath")
+TRAVIS_PULL_REQUEST = os.getenv("TRAVIS_PULL_REQUEST","false")
+TRAVIS_PULL_REQUEST_TITLE = os.getenv("TRAVIS_PULL_REQUEST_TITLE","")
+TRAVIS_PULL_REQUEST_LINK = os.getenv("TRAVIS_PULL_REQUEST_LINK","")
+TRAVIS_COMMIT = os.getenv("TRAVIS_COMMIT","")
+GITHUBTOKEN = os.getenv("GITHUBTOKEN","")

```

Comment:
I think it would be better off as a single function in the existing `WebpageFunctions.py`, so it avoids creating another python file. I think it might also make sense then to rename `WebpageFunctions.py` to `Common.py`, which I believe is convention.

Reply:
I have made a change to have it structured as a class in common.py! I think it makes sense as we can have more common classes is needed, preserved that they are "WebPageFunctions" and can also initialise the env variables upon class initialisation. (This can also change to initialising from a config file if we decide to move away from env vars)